test/tstKokkosToolsDistributedAnnotations.cpp

```diff
@@ -87,7 +87,7 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(
                     isPrefixedWith(label, "ArborX::TreeTraversal::nearest::") ||
                     isPrefixedWith(label, "ArborX::CrsGraphWrapper::") ||
                     isPrefixedWith(label, "ArborX::Sorting::") ||
-                    isPrefixedWith(label, "Kokkos::SortImpl::") ||
+                    isPrefixedWith(label, "Kokkos::") ||
```

Comment:
What other label was there?

Reply:
`Kokkos::Experimental::SYCL::USMObjectMem`

Reply:
Is there a reason to check anything other than `Kokkos::` and `ArborX::`?

Reply:
Well if we can narrow down what gets called from Kokkos it is good but I think Daniel's suggestion is fine